app/services/external_apis/base_service.rb

```diff
@@ -114,7 +114,7 @@ def options(additional_headers: {}, debug: false)
           headers: headers.merge(additional_headers),
           follow_redirects: true
         }
-        hash[:debug_output] = STDOUT if debug
+        hash[:debug_output] = $stdout if debug
```

Comment:
Cool! I'd not seen the `STDOUT` `$stdout` difference before.

Reply:
we either until Rubocop flagged it ;)